kong/plugin_service_test.go

```diff
@@ -107,6 +107,36 @@ func TestPluginWithTags(T *testing.T) {
 	assert.NoError(err)
 }
 
+func TestPluginWithOrdering(T *testing.T) {
+	RunWhenEnterprise(T, ">=3.0.0", RequiredFeatures{})
+	assert := assert.New(T)
+
+	client, err := NewTestClient(nil, nil)
+	assert.NoError(err)
+	assert.NotNil(client)
+
+	plugin := &Plugin{
+		Name: String("request-termination"),
+		Ordering: &PluginOrdering{
+			Before: PluginOrderingPhase{
```

Comment:
Can we also add a test using `After` and one using both?

Reply:
9723ed5 just does a single one with both. For the successful case we just care that all the fields marshal and unmarshal to/from the admin API; the nil versions aren't of much interest.